libraries/AP_Mount/AP_Mount_Backend.cpp

```diff
@@ -118,6 +118,34 @@ void AP_Mount_Backend::set_rate_target(float roll_degs, float pitch_degs, float
     }
 }
 
+void AP_Mount_Backend::control_mount_open_servo() {
+    // check if the open function is assigned to a servo channel.
+    uint8_t open_chan;
+    if (!SRV_Channels::find_channel(SRV_Channel::k_mount_open, open_chan) &&
+        !SRV_Channels::find_channel(SRV_Channel::k_mount2_open, open_chan)) {
+        return; // open function not assigned, do nothing.
+    }
+
+    // get the configured PWM values for open and closed states.
+    SRV_Channel *channel = SRV_Channels::srv_channel(open_chan);
+    if (!channel) {
+        return; // channel not found.
+    }
+    uint16_t pwm_open = channel->get_output_max();
+    uint16_t pwm_closed = channel->get_output_min();
+
+    // determine the desired open state based on the current mode.
+    bool mount_open = true;
```

Comment:
this section could be made more concise although it might not actually make the final flash size any smaller.

I wonder if you could put "const" before the local variables that won't change?  e.g. "const uint16_t pwm_open = ..."

.. and also maybe use the terniary operator for mount_open similar to what was in the original code (but use true and false instead of "0" and "1" which really wasn't good in the original code.